test/blocks/marketo/marketo.test.js

```diff
@@ -21,76 +21,56 @@ describe('marketo', () => {
     expect(el.style.display).to.equal('none');
   });
 
-  it('init marketo form', async () => {
-    setConfig(config);
-    document.body.innerHTML = innerHTML;
-    const el = document.querySelector('.marketo');
-    init(el);
-
-    const wrapper = await waitForElement('.marketo-form-wrapper');
-    expect(wrapper).to.exist;
-
-    const title = el.querySelector('.marketo-title');
-    expect(title).to.exist;
-  });
-
-  it('load marketo disables stylesheets', async () => {
-    const formRow = await waitForElement('.mktoFormRow');
-    expect(formRow).to.exist;
-
-    const styleSheets = [...document.styleSheets];
-    const styleSheet = styleSheets.find((sheet) => sheet.href === `http:${config.marketoBaseURL}/js/forms2/css/forms2.css`);
-    expect(styleSheet.disabled).to.be.true;
-  });
-
-  it('marketo hidden fields', async () => {
-    const hiddenInput = await waitForElement('input[name="hiddenField"]');
-    expect(hiddenInput).to.exist;
-  });
-
-  it('validate marketo fields error', async () => {
-    const error = await waitForElement('.marketo-error');
-    const errorMessage = 'There are some fields that require your attention';
+  describe('with correct config', () => {
+    before(() => {
+      setConfig(config);
+      document.body.innerHTML = innerHTML;
+      const el = document.querySelector('.marketo');
+      init(el);
+    });
 
-    expect(error).to.exist;
+    it('init marketo form', async () => {
+      const wrapper = await waitForElement('.marketo-form-wrapper');
+      expect(wrapper).to.exist;
 
-    expect(window.MktoForms2).to.exist;
-    const form = window.MktoForms2.getForm(config.marketoFormID);
+      const title = document.querySelector('.marketo-title');
+      expect(title).to.exist;
+    });
 
-    formValidate(form, false, error, errorMessage);
-    expect(error.classList.contains('alert')).to.be.true;
-  });
-
-  it('marketo field error visible', async () => {
-    const button = await waitForElement('.marketo button');
-    button.click();
+    it('marketo hidden fields', async function () {
```

Comment:
nit: `it` strings should start with a verb

Reply:
Done